src/components/signup-view/signup-view.jsx

```diff
@@ -44,59 +49,99 @@ export const SignupView = () => {
   };
 
   return (
-    <form onSubmit={handleSubmit} noValidate>
-      <label htmlFor="username">
-        Username:*
-        <input
-          type="text"
-          id="username"
-          value={username}
-          onChange={(e) => setUsername(e.target.value)}
-          required
-          minLength={5}
-          pattern="^[a-zA-Z0-9]*$"
-          onInvalid={(e) => {
-            if (e.target.validity.patternMismatch) {
-              e.target.setCustomValidity(
-                "Username can only use alphanumeric characters."
-              );
-            } else {
-              e.target.setCustomValidity("");
-            }
-          }}
-        />
-      </label>
-      <label htmlFor="email">
-        E-Mail:*
-        <input
-          type="email"
-          id="email"
-          value={email}
-          onChange={(e) => setEmail(e.target.value)}
-          required
-        />
-      </label>
-      <label htmlFor="birthday">
-        Birthday:
-        <input
-          type="date"
-          id="birthday"
-          value={birthday}
-          onChange={(e) => setBirthday(e.target.value)}
-        />
-      </label>
-      <label htmlFor="password">
-        Password:*
-        <input
-          type="password"
-          id="password"
-          value={password}
-          onChange={(e) => setPassword(e.target.value)}
-          required
-          minLength={8}
-        />
-      </label>
-      <button type="submit">Submit</button>
-    </form>
+    <div class="p-4 rounded-4 bg-body">
+      <h1 class="text-center">Sign up</h1>
```

Comment:
Could you please replace all instances of `class` with `ClassName`.  React yells about this in console.
<img width="751" alt="Screenshot 2023-06-26 at 10 23 53 PM" src="https://github.com/KatGaertner/myFlix/assets/29123354/baafe17a-9089-47e8-b14c-daa5cbbe4891">

Reply:
Sorry! I had replaced them for a later version, but I forgot to double-check here.